.ci/jenkins/Jenkinsfile.daily

```diff
@@ -106,6 +106,17 @@ pipeline {
                 }
             }
         }
+        stage('Deploy process-migration-service locally with jdk11'){
+            steps {
+                dir("${WORKSPACE}/bc/kiegroup_process_migration_service") {
+                    configFileProvider([configFile(fileId: '771ff52a-a8b4-40e6-9b22-d54c7314aa1e', targetLocation: 'jenkins-settings.xml', variable: 'SETTINGS_XML_FILE')]) {
+                        withEnv(["JAVA_HOME=${tool 'kie-jdk11'}", "PATH=${tool 'kie-jdk11'}/bin:${env.PATH}"]) {
```

Comment:
sorry, I missed that part from here, why don't you use this approach https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/e469889a614b32fc4e94030e4f799248763e1287/.ci/jenkins/Jenkinsfile.buildchain#L79? I think it's more clear, something like
```
stage('Deploy process-migration-service locally with jdk11'){
    tools {
      jdk 'kie-jdk11'
    }
    steps {
        dir("${WORKSPACE}/bc/kiegroup_process_migration_service") {
            configFileProvider([configFile(fileId: '771ff52a-a8b4-40e6-9b22-d54c7314aa1e', targetLocation: 'jenkins-settings.xml', variable: 'SETTINGS_XML_FILE')]) {
                  sh 'mvn -B -e -U clean deploy -Dfull -Drelease -DaltDeploymentRepository=local::default::file://$WORKSPACE/deploy-dir -s $SETTINGS_XML_FILE -Dkie.maven.settings.custom=$SETTINGS_XML_FILE -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true -Dgwt.memory.settings="-Xmx10g" --clean-up-script="$WORKSPACE/clean-up.sh"'
            }
        }
    }
}
```

Reply:
but i have still the same question if we need to add this stage and do not let it be without building? no answer yet